Casks/w/waveterm.rb

```diff
@@ -0,0 +1,25 @@
+cask "waveterm" do
+  version "0.5.1"
+  sha256 "fe965fe581a66bbd7c394dbfd7ebb1635cd0bf46329042ef86db123fbd7d5751"
+
+  url "https://dl.waveterm.dev/builds/waveterm-macos-universal-v#{version}.dmg"
+  name "Wave Terminal"
+  name "WaveTerm"
+  desc "Modern Terminal for Seamless Workflows"
```

Comment:
```suggestion
  desc "Terminal emulator"
```

You are welcome to suggest an alternative, but it should adhere to the `[guidelines](https://docs.brew.sh/Cask-Cookbook#stanza-desc)`.

Reply:
got it.  missed those guidelines when writing the desc, will update.